### PR TITLE
Vm group status

### PR DESCRIPTION
**Pull request by KostiaLevchenko**
- added field status to VMGroup model
- implemented vmg status for removing vm group
- implemented vmg get status
- implemented vmg status for creating vm group

### CODE REVIEW

File: kubehub/models/vm_group.py [hunk 1]
```diff
@@ -8,9 +8,10 @@ class VMGroup(models.Model):
     name = models.CharField(max_length=name_max)
     user_id = models.IntegerField()
     readonly_fields = ('name', 'user_id')
+    status = models.CharField(max_length=name_max)
```
Comment (kopachevsky) [inline kubehub/models/vm_group.py]: add choices field to restrict status only to enum listed values

File: kubehub/views/vm_group_view.py [hunk 2]
```diff
@@ -21,32 +21,85 @@ def vm_group_list(request):
...
+        return model_to_dict(vmg)
+
+
 @csrf_exempt
 def vm_group_remove(request):
     if request.method == 'POST':
         try:
             data = json.loads(request.body)
             pk = data.get('vm_group_id')
+            status_update(pk, "removing")
```
Comment (kopachevsky) [inline kubehub/views/vm_group_view.py:103]: how long it will take for end user to get response? 10 mins?

Comment (KostiaLevchenko) [reply kubehub/views/vm_group_view.py:103]: user will get "removing" status immediately, approximately 10 minutes will be needed to delete group from proxmox (can't use multithreading for this coz proxmox can't handle multiple delete requests)

File: kubehub/models/vm_group.py (hunk 1 above)
Comment (KostiaLevchenko) [reply kubehub/models/vm_group.py]: fixed in a3a1558

File: kubehub/urls.py [hunk 3]
```diff
@@ -9,6 +9,7 @@
     url(r'^edit$', crud_views.cloud_provider_edit, name='cloud_provider_edit'),
     url(r'^cluster/create$', kubespray_deploy.kubespray_deploy, name='cluster_create'),
     url(r'^vm/group/list$', vm_group_view.vm_group_list, name='virtual_machines_group_list'),
+    url(r'^vm/group/get/status$', vm_group_view.get_vm_group_status, name='vm_group_status'),
```
Comment (kopachevsky) [inline kubehub/urls.py]: why /get/status not just /status ??

**Review by kopachevsky — APPROVED** [review]

File: kubehub/urls.py (hunk 3 above)
Comment (KostiaLevchenko) [reply kubehub/urls.py]: fixed in e402831

**Comment by KostiaLevchenko** [comment]
@kopachevsky should we add error field to vmg model to store error message if it arose ?

**Pull request merged**